KatyaBebeshko/HW1/task2.py

```diff
@@ -0,0 +1,24 @@
+a = 789
+b = 367
+
```

Comment:
good job
try to use f string for output information like this:
print(f"a+b={a+b}") or print(f"sum={a+b}")